### ISSUE TITLE
损失函数符号问题

### ISSUE THREAD

**Issue by lsquser** [jack57lee/thumt#4]
请问在实现代码时，实现的是负的余弦相似度吗
代码里cos_diff = tf.reduce_mean(cos_diff, axis=[-2,-1]) + 1.0最后加的1是什么意思。
cos_diff是表示加了负号的吗，还是没加负号的。论文里面是添加了负号的呀
